source/stylesheets/site.css.scss

```diff
@@ -1579,33 +1579,18 @@ ul.showcase {
 
   width: 33.333%;
   margin: 10px 0;
-  &:nth-child(3n+1) {
-    a {
-      margin: 0 auto 0 0;
-    }
-  }
```

Comment:
If it's up to me, I'm ok with losing left-hand and right-hand margin alignment, as it is still pleasantly centered. It is possible to accomplish the alignment, but Im not terribly motivated to, so if someone is...